deals-add.js

```diff
@@ -0,0 +1,223 @@
+var script = document.createElement('script');
+script.type = 'text/javascript';
+script.src = 'https://s.cdpn.io/3/clamp.js';
+document.head.appendChild(script);
+
+var script = document.createElement('script');
```

Comment:
Also, I highly suggest not adding remote scripts this way. Better to just add as html in webflow
i.e. add this at the bottom, but before we fetch THIS script. If you're agreeable I'll do it for you in webflow
`<script async src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.20.1/moment-with-locales.js" type="text/javascript"></script>`